clients/pkg/promtail/targets/journal/journaltarget.go

```diff
@@ -264,6 +269,7 @@ func (t *JournalTarget) formatter(entry *sdjournal.JournalEntry) (string, error)
 		bb, err := json.Marshal(entry.Fields)
 		if err != nil {
 			level.Error(t.logger).Log("msg", "could not marshal journal fields to JSON", "err", err)
```

Comment:
do we have access to the unit here? could we include it in the log message since we're not going to include it in the metric?

Reply:
Added it, its not always available, but I'd say most of the time it is & adds some value